DirectProgramming/DPC++FPGA/ReferenceDesigns/cholesky_inversion/src/cholesky_inversion.hpp

```diff
@@ -0,0 +1,132 @@
+#ifndef __CHOLESKY_INVERSION_HPP__
```

Comment:
a general question: why do we put the FPGA device code in a header file instead of a separate `.cpp` file? Isn't there a benefit to being able to re-compile the testbench independently of the device code?